app/views/accounts/connected_accounts/show.html.erb

```diff
@@ -1,3 +1,5 @@
+<% title t('account.navigation.connected_accounts') %>
+
 <h1 class="mt0">
   <%= t('headings.account.connected_accounts') %>
```

Comment:
Thinking out loud a bit: It's sensible that the page's title and the page's `h1` are the same, which raises a question whether it's worth having duplicate strings for these. And if we do have duplicate strings (one for the navigation and one for the page title), whether the value we use in `title` should be the same as the page's `h1`, or the same as what's shown in the navigation.

Reply:
Yeah, 100% agree